hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/store/contracts/WorldLedgersTest.java

```diff
@@ -1015,6 +1051,10 @@ private void setUpToken(final MerkleToken token) throws DecoderException {
         token.setTokenType(FUNGIBLE_COMMON);
     }
 
+    private void tokenAccessor() {
```

Comment:
Do we need a special method for initializing the `tokenAccessor` on a single line? I believe in this case, readibility would be improved if we directly initialize the `tokenAccessor`, without defining and calling a separate method.